project.clj

```diff
@@ -6,7 +6,8 @@
   :jvm-opts ["-Djava.awt.headless=true"]
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [org.clojure/tools.cli "0.3.5"]
-                 [net.mikera/imagez "0.10.0"]]
+                 [net.mikera/imagez "0.10.0"]
+		 [com.taoensso/tufte "1.0.2"]]
```

Comment:
you seem to be using tab characters, which i disapprove of, and you also seem to be using 8.5-space tabs, which i've got no problem with